README.md

```diff
@@ -19,17 +19,15 @@ Wil je een aansluiting ontwikkelen? Dan is de [getting started](./docs/getting-s
 https://eu-rm.roadmunk.com/publish/29a13c572a0dfc79f1d0386fc572e15383afe11f
 
 ## Documentatie
-* [Productvisie](https://github.com/VNG-Realisatie/BRK-bevragingen/blob/master/docs/productvision.md)
-* [Definition of Ready](https://github.com/VNG-Realisatie/BRK-bevragingen/blob/master/docs/definition_of_ready.md)
-* [Definition of Done](https://github.com/VNG-Realisatie/BRK-bevragingen/blob/master/docs/definition_of_done.md)
-* Hoe u kunt [bijdragen](https://github.com/VNG-Realisatie/Tutorial/blob/master/CONTRIBUTING.md)
-* [Omgangsvormen](https://github.com/VNG-Realisatie/Tutorial/blob/master/CODE_OF_CONDUCT.md)
+* [Productvisie](https://github.com/VNG-Realisatie/BRK-bevragingen/blob/v1.1.0/docs/productvision.md)
```

Comment:
waarom gebruiken we hier geen relatieve links?

Dus in plaats van https://github.com/VNG-Realisatie/BRK-bevragingen/blob/master/docs/productvision.md
relatieve link ../docs/productvision.md